src/main/java/com/kainos/ea/daos/RoleDao.java

```diff
@@ -15,44 +15,60 @@
 import java.util.List;
 
 public class RoleDao {
-    /**
-     * Gets.
-     * @return Returns list of open job roles
-     * @throws SQLException
-     * @throws FailedToRetrieveException
-     */
+
     public List<RoleResponse> getAllJobRoles()
             throws SQLException, FailedToRetrieveException {
+        return getOrderedJobRoles(null, null);
```

Comment:
It looks like we've no need to create a new function here for ordering, when it's simply just being called by this function. I'd just edit the getAllJobRoles function instead.

Reply:
changed just oine function